samples/sqltransport/store-and-forward/sample.md

```diff
@@ -68,13 +69,16 @@ NOTICE: Other properties of a message (such as defer time) are not captured in t
 
 In the receive pipeline the new behavior is placed just before loading the message handlers.
 
-snippet:OutboxLoopbackReceiveBehavior
+snippet:ForwardBehavior
```

Comment:
Can we add note that in this example DTC is still needed? And that this dependancy can be removed by suppressing `TransactionScope` in `ForwardBehavior` but will result in at-leat-once message delivery to the receiver.